server/repository/src/migrations/v1_06_00/name_is_deceased.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{migrations::sql, StorageConnection};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    if cfg!(feature = "postgres") {
+        sql!(
+            connection,
+            r#"
+            UPDATE name set is_deceased = false WHERE is_deceased is null;
+            ALTER TABLE name ALTER COLUMN is_deceased SET NOT NULL;
+            ALTER TABLE name ALTER COLUMN is_deceased SET DEFAULT false;
+          "#
+        )?;
+    }
+
+    // the same as above - but there's no alter column for SQLite
+    if cfg!(not(feature = "postgres")) {
```

Comment:
this is the overkill version. not needed? we could just have some installs with a nullable bool and others without?